examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java

```diff
@@ -89,7 +89,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     server.blockUntilShutdown();
   }
 
-  private class GreeterImpl extends GreeterGrpc.GreeterImplBase {
+  static class GreeterImpl extends GreeterGrpc.GreeterImplBase {
```

Comment:
Showing how to test a service seems pretty natural. Yes, this is a full server and not just the service, but testing them separately is non-invasive here and would mirror what most applications would need. And yes, this service is trivial, but "going through the motions" seems appropriate.